org-roam.el

```diff
@@ -458,6 +475,7 @@ Valid states are 'visible, 'exists and 'none."
 (defun org-roam ()
   "Pops up the window `org-roam-buffer' accordingly."
   (interactive)
+  (setq org-roam-last-window (get-buffer-window))
```

Comment:
I think this may not be intuitive, and maybe `org-roam-last-window` should be set to the last `org-roam` file being interacted with, but for now I think this is fine and should be equivalent 99% of the time, we'll see if people complain.